src/extension/commandPaletteHandler.ts

```diff
@@ -96,7 +96,8 @@ export class CommandPaletteHandler {
 
         return this.reactNativePackager.start()
             .then(() => {
-                return new CommandExecutor(this.workspaceRoot).spawnReactCommand(command, args, null);
+                return new CommandExecutor(this.workspaceRoot).spawnReactCommand(command,
```

Comment:
This method is called in multiple instances, by both the Android and iOS platforms. Did you verify that all the scenarios still work as expected?